feat: add others cookbooks (#267)
agno-agi/agno-docs: examples/concepts/others/instructions.mdx

---
title: Instructions
---

> This example shows how to set the instructions of an agent.

Reviewer: ```suggestion
This example shows how to provide specific instructions to an agent.
```
